src/hooks/use-add-site.ts

```diff
@@ -113,13 +113,6 @@ export function useAddSite() {
 						};
 					}
 
-					if ( newSite.wpVersion !== wpVersion ) {
```

Comment:
Everything seems to be working correctly, creating new sites and editing sites.
So removing the test `should update site with WordPress version after creation` makes sense.